### PR TITLE
[enocean] Addon suggestion finder xml

### PR DESCRIPTION
**Pull request by andrewfg**
This PR adds the suggestion finder xml to the addon.xml file in anticipation of the [PR in openhab-core](https://github.com/openhab/openhab-core/pull/3922) being merged.

Signed-off-by: Andrew Fiddian-Green <software@whitebear.ch>

### CODE REVIEW

**Comment by andrewfg** [comment]
Note: since https://github.com/openhab/openhab-core/pull/3922 has been merged, this one is now ready for review.

**Comment by andrewfg** [comment]
@fruggy83 does this binding work with remote ip connected sticks according to RFC221?

File: bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/addon/addon.xml [hunk 1]
```diff
@@ -8,4 +8,20 @@
...
 	<connection>local</connection>
 
+	<discovery-methods>
+		<discovery-method>
+			<service-type>usb</service-type>
+			<match-properties>
+				<match-property>
+					<name>remote</name>
+					<regex>false</regex>
+				</match-property>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/addon/addon.xml:18]: Just for my understanding, what does this criterai mean exactly ?

Comment (andrewfg) [reply bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/addon/addon.xml:18]: There are two types of USB serial 'stick' devices -- namely those which are physically connected locally to the PC, and those which are connected remotely from another PC via RFC2217 'socat'. Some bindings (such as ZigBee and Z-Wave) can use both. But I am not sure if this binding EnOcean can use the latter, which is why we currently exclude it from the suggestion finder. However if @fruggy83 can confirm that it also works via remote, then we can remove this filter.

Comment (lolodomo) [reply bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/addon/addon.xml:18]: Ok, understood.

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Pull request merged**